.github/workflows/lecture_participation.yml

```diff
@@ -0,0 +1,36 @@
+name: Lecture Participation Information
+
+on:
+  issue_comment:
+    types: [created]
+
+permissions:
+  issues: write
+
+jobs:
+  track-participation:
+    if: github.event.issue.title == 'Lecture participation tracker 2024'
```

Comment:
I thought about how to implement this. One could do it as in [assignment-statistics.yml](https://github.com/KTH/devops-course/blob/87ecf8925262aab78ed8098b1438abadd0a2ee1e/.github/workflows/assignment-statistics.yml), where it's set as an environmental variable and then checked in the python script. The problem with that is that it will unnecessarily continue the CI runner, set up Python, install dependencies etc., even if it's not the correct issue. And I suppose changing a number in the workflow is not more work than changing a repo secret.

So I'll probably just do

`if: github.event.issue.number == 2370`

Reply:
Addressed in commit a0a9a86

Reply:
I am okay with this approach.